service/mapper/mapper.go

```diff
@@ -160,39 +162,58 @@ Outer:
 		// if no change to the state trie happens between multiple blocks, at
 		// which point we map the second and any subsequent blocks without
 		// change to an empty set of deltas.
-		hash := m.trie.RootHash()
+
+		commitTree := tree.RootHash()
+		var log zerolog.Logger
 	Inner:
 		for {
 
+			log = m.log.With().
+				Uint64("height", height).
+				Hex("commit_trie", commitTree).
+				Int("num_deltas", len(deltas)).
+				Logger()
+
 			// We first try to get the next commit by height, because that is
 			// the sign that the block has been sealed. If the retrieval times
 			// out, we loop right back into this condition, because it means the
 			// network might be stalling. If the error indicates we finished,
 			// then we reached the end of the WAL and can finish without error.
-			commit, err := m.chain.Commit(height)
+			commitHeight, err := m.chain.Commit(height)
```

Comment:
I'm not fond of the naming here. I'd expect `commitHeight` to be a `uint64` from the name. WDYT of calling it `nextCommit` or something similar instead?